RealtimeKafkaToHBase/config.json

```diff
@@ -0,0 +1,21 @@
+{
+  "template": "etlRealtime",
+  "config": {
+    "source": {
+      "name": "Kafka",
+      "properties": {
+        "kafka.partitions": 10,
```

Comment:
@sreevatsanraman Do we need to explain what these properties mean in the README.rst? May be a line at least?